### DISCUSSION TITLE
2 rfid readers?

### DISCUSSION THREAD

**Discussion by edednedy** [MiczFlor/RPi-Jukebox-RFID#2282] [Q&A]
Would it be possible to hook-up two RC522 Modules to one Rapberry Pi running RFID Jukebox?
I want to be able to scan a card on either the left or the right side of a box, and have either one read the card and start playback. I don't need to identify which one is scanning and I don't want to have to turn them on and off. I just want either one to read regardless of which one is scanned. Would it be possible to just wire them in parallel?

**Comment by s-martin** [comment]
Version 2 has the feature to use two RFID readers (select Multi during installation).

But I don’t think it’s possible to use two RC522 at the same time as right now there are no two independent wirings possible.
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Wiring_for_RC522_card_reader

It’s possible to use an RC522 and a USB reader, for example. Check, if RFID cards work for both readers.

maybe you can explain your use case a little bit more.

**Comment by edednedy** [comment]
I want to put the cards inside record albums. So the card will be located in the bottom left corner (cover facing up). But I also want to be able to place the album "back side up" (So I can see the track listing). This will put the card in the lower right position. I want to be able for it to read the card in either position. I can't put it in the middle because the cards won't stay and I don't want to damage the covers with stickers.

**Comment by edednedy** [comment]
About halfway down this post: https://raspberrypi.stackexchange.com/questions/137059/how-to-connect-multiple-rfid-readers-rc522-to-a-rpi
Somebody does this by wiring everything in parallel except the reset pin, he says it works on five at a time. I only need two, but wouldn't the code have to be revised in order for it to work in RFID Jukebox?
What would happen if you hooked all of them in parallel including the RST pin?

**Reply by s-martin** [reply]
Interesting 😀

From the description it looks like your desired use case could work.

If I understand your use case correctly you don't want to preselect a reader, but want to use both simultaneously.

If you already have both readers I would just try it out, while enabling both readers at once via RST.

I'm really interested, if this works :)

**Reply by AlvinSchiller** [reply]
Sadly, using the exact same pins for two rc522 readers does not work.

I tested this with two readers connected via a GPIO extension board like [this](https://www.az-delivery.de/en/products/gpio-expansion-board-fur-raspberry-pi) one. The first reader ("closer" to the pi pins)  recognizes all card scan like normal, the second in line does not recognize anything.
If the readers are connected alone, both work fine.
